org.alloytools.alloy.core/src/main/java/org/alloytools/graphics/util/AlloyGraphics.java

```diff
@@ -46,12 +50,35 @@ public static synchronized String matchBestFontName(String fontNames) {
                 if (font != null)
                     return font.getFontName();
             } else {
-                if (availableFontNames.contains(name))
+                String soundex = toSoundex(name);
+                String fontName = availableFontNames.get(soundex);
+                if (fontName != null)
                     return name;
             }
         }
         return names[names.length - 1];
     }
 
+    private static String toSoundex(String name) {
+        StringBuilder sb = new StringBuilder(name);
+        for (int i = 0; i < sb.length(); i++) {
```

Comment:
instead of mutating `sb` while iterating over it (which is kind of sketchy), I'd rather iterate over `name` and decide which chars to add to `sb` and which ones to skip